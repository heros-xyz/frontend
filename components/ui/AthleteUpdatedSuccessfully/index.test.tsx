import { render } from "@testing-library/react";
import mockRouter from "next-router-mock";
import AddSportProfileSuccessfully from ".";

jest.mock("next/router", () => require("next-router-mock"));

it("renders add Add Sport Profile Successfully unchanged", () => {
  mockRouter.push("/initial-path");
  const { container } = render(<AddSportProfileSuccessfully />);
  expect(container).toMatchSnapshot();
});
