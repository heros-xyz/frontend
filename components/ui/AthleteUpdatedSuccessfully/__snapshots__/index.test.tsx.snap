// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders add Add Sport Profile Successfully unchanged 1`] = `
<div>
  <div
    class="css-15p2wse"
  >
    <div
      class="css-1kmmdcq"
    >
      Basic information
    </div>
    <div
      class="chakra-container css-17f3xev"
    >
      <div
        class="css-1ispe57"
      >
        <div
          class="css-aj1n7q"
        >
          <div
            class="css-k38zka"
          >
            <svg
              class="chakra-icon css-tve76j"
              focusable="false"
              viewBox="0 0 91 90"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                clip-path="url(#clip0_5448_58627)"
              >
                <path
                  d="M83.1215 87.5921C85.2121 86.9642 90.0205 85.9177 90.0205 83.5107C90.125 74.1968 88.6616 42.1735 86.8846 34.6386C85.1076 26.7898 57.6163 -3.97764 51.6581 0.417705C45.909 4.81305 58.6616 23.3363 58.6616 23.3363C58.6616 23.3363 43.1912 10.9875 38.2783 16.534C33.2609 22.2898 48.1041 34.0107 48.6268 35.3712C48.6268 35.3712 33.2609 26.1619 31.0658 32.0224C29.6024 35.8945 32.1111 38.6154 41.8323 44.2665C47.7905 47.72 42.8776 80.7898 42.9822 86.441C42.8776 91.0456 72.4595 91.0456 83.1215 87.5921Z"
                  fill="currentColor"
                />
                <path
                  d="M72.5649 24.8015C66.3977 24.1736 53.6451 24.8015 51.0318 38.1969C47.8959 54.3132 62.321 53.5806 62.5301 45.7318C62.7391 36.8364 76.4325 37.569 77.0597 44.3713"
                  stroke="black"
                  stroke-miterlimit="10"
                  stroke-width="1.5"
                />
                <path
                  d="M57.7219 72.8367C53.7497 69.4878 52.0773 65.9297 51.7637 60.9064C51.1365 52.4297 61.4849 49.4994 63.5755 55.8832C64.9344 59.9646 67.4431 63.2087 70.1609 63.6274C70.1609 63.6274 77.8961 64.2553 76.6417 57.139"
                  stroke="black"
                  stroke-miterlimit="10"
                  stroke-width="1.5"
                />
                <path
                  d="M9.95173 20.3021C10.6834 17.4765 10.2653 16.2207 11.6242 16.2207C12.774 16.2207 12.3559 17.7905 12.8786 19.4649C13.6103 22.0812 14.4465 25.1161 20.4047 25.8486C23.4361 26.2672 22.8089 27.7323 21.1364 28.0463C20.1956 28.2556 20.1956 28.1509 18.7322 28.3602C13.8193 29.3021 13.1921 33.5928 12.6695 36.837C12.3559 38.7207 10.997 38.7207 10.5789 36.2091C10.0563 33.4881 10.1608 28.6742 2.21654 28.2556C0.544062 28.1509 0.544062 26.7905 1.79842 26.4765C5.77054 25.5347 8.48831 26.1626 9.95173 20.3021Z"
                  fill="#E2FF65"
                  stroke="black"
                  stroke-miterlimit="10"
                  stroke-width="1.5"
                />
                <path
                  d="M20.091 32.4402C20.5092 30.7658 20.3001 30.0332 21.0318 30.0332C21.659 30.0332 21.4499 30.9751 21.7635 32.0216C22.1816 33.5913 22.7043 35.3704 26.2583 35.789C28.0353 35.9983 27.7217 36.9402 26.6764 37.1495C26.1538 37.2541 26.1538 37.1495 25.213 37.3588C22.2862 37.882 21.9726 40.4983 21.659 42.382C21.4499 43.5332 20.7182 43.5332 20.4046 42.0681C20.091 40.4983 20.1956 37.5681 15.4917 37.3588C14.551 37.3588 14.551 36.5216 15.2827 36.3123C17.5823 35.4751 19.1503 35.8937 20.091 32.4402Z"
                  fill="#E2FF65"
                  stroke="black"
                  stroke-miterlimit="10"
                  stroke-width="1.5"
                />
                <path
                  d="M25.9444 21.0346C26.258 19.8834 26.0489 19.4648 26.5716 19.4648C26.9897 19.4648 26.8851 20.0927 27.0942 20.7207C27.4078 21.7672 27.7214 22.9183 30.021 23.2323C31.1709 23.4416 30.9618 23.9648 30.3346 24.0695C29.9165 24.1741 30.021 24.0695 29.3939 24.1741C27.5123 24.4881 27.3033 26.1625 27.0942 27.4183C26.9897 28.1509 26.467 28.1509 26.258 27.209C26.0489 26.1625 26.0489 24.2788 23.0175 24.1741C22.3904 24.1741 22.3904 23.6509 22.913 23.4416C24.2719 23.1276 25.3172 23.3369 25.9444 21.0346Z"
                  fill="#E2FF65"
                  stroke="black"
                  stroke-miterlimit="10"
                  stroke-width="1.5"
                />
              </g>
              <defs>
                <clippath
                  id="clip0_5448_58627"
                >
                  <rect
                    fill="white"
                    height="90"
                    transform="translate(0.125)"
                    width="90"
                  />
                </clippath>
              </defs>
            </svg>
          </div>
        </div>
        <div
          class="css-14s5ckc"
        >
          <div
            class="css-1s2o82u"
          >
            <div
              class="css-1uoge8t"
            >
              <div
                class="css-1so8xn5"
              >
                <p
                  class="chakra-text css-xevqdv"
                >
                  Updated Successfully
                </p>
                <p
                  class="chakra-text css-5wf9xv"
                >
                  Your changes have been successfully saved.
                </p>
              </div>
              <div
                class="css-e26nma"
              >
                <button
                  class="chakra-button css-k6pd2s"
                  type="button"
                >
                  GO TO CHECKLIST
                  <span
                    class="css-9n3bbc"
                  >
                    <svg
                      fill="none"
                      height="14"
                      viewBox="0 0 15 14"
                      width="15"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M1.25 7L13.75 7M13.75 7L8.125 1.375M13.75 7L8.125 12.625"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                        stroke-width="1.5"
                      />
                    </svg>
                  </span>
                </button>
              </div>
            </div>
          </div>
          <button
            class="chakra-button css-1mv50kv"
            type="button"
          >
            GO TO CHECKLIST
            <span
              class="css-9n3bbc"
            >
              <svg
                fill="none"
                height="14"
                viewBox="0 0 15 14"
                width="15"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M1.25 7L13.75 7M13.75 7L8.125 1.375M13.75 7L8.125 12.625"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="1.5"
                />
              </svg>
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
